CHANGELOG.md

```diff
@@ -1669,3 +1670,4 @@
 [@eagletmt]: https://github.com/eagletmt
 [@apiology]: https://github.com/apiology
 [@alexdowad]: https://github.com/alexdowad
+[@alexdowad]: https://github.com/alexdowad
```

Comment:
There is already a link to your account in the `CHANGELOG`. Adding a second link is not necessary.